### PR TITLE
examples: Add examples

### PR DESCRIPTION
**Pull request by mrzelak**
Add visualisation examples for P9, P10 and P16. Update README.md section about running examples.

### CODE REVIEW

File: src/example/test_border.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    edges.append(Edge(4,11,EdgeAttrs(kind='q', flag=False, handle=38)))
+    edges.append(Edge(4,12,EdgeAttrs(kind='q', flag=False, handle=39)))
+    graph.add_node_collection(nodes)
+    graph.add_edge_collection(edges)
+
+    graph.display()
+    plt.show()
+
+    P1()(graph)
+
```
Comment (kkafar) [inline src/example/test_border.py:65]: What is this file for?

File: README.md [hunk 2]
```diff
@@ -52,3 +52,14 @@ https://docs.google.com/document/d/1aBiGuepeNUuPSGl8aK_Ilit3hJedHvKwng_xBeh43OY/
...
+
+Run examples with visualisation:
+
+```bash
+cd src
+export PYTHONPATH=`pwd`
+cd example
+python main.py <arg>
+```
+Where <arg> can be 1, 10, 16, 169, border.
```
Comment (kkafar) [inline README.md:65]:
```suggestion
```

This will age better w/o specifying numbers here.

**Review by kkafar — COMMENTED** [review]
Hey, it will be much better if we indeed move all the examples to `src/example` module, separate file for each example, but let's keep single main in `src/` directory. This approach won't require tricks with python path etc.

File: src/example/test_border.py (hunk 1 above)
Comment (mrzelak) [reply src/example/test_border.py:65]: This was a test for visualizing the problem with edge_equality. Now it's basically useless.

**Review by kkafar — APPROVED** [review]
Great! Thanks for taking account of my suggestions

**Pull request merged**